qa/tasks/cephfs/test_quota.py

```diff
@@ -160,3 +160,37 @@ def test_disable_enable_human_readable_quota_values(self):
         self.mount_a.setfattr("./subdir", "ceph.quota.max_bytes", "0M")
         self.assertEqual(self.mount_a.getfattr("./subdir",
                                                "ceph.quota.max_bytes"), None)
+        
+    def test_human_readable_float_quota_values(self):
+        """
+        test human-readable float values for setting ceph.quota.max_bytes
+        """
+        self.mount_a.run_shell(["mkdir", "subdir"])
+
+        self.assertEqual(self.mount_a.getfattr("./subdir",
+                                               "ceph.quota.max_bytes"), None)
+
+        readable_values = {"1.5K": "1536",
+                           "3.6Ki": "3686",
```

Comment:
one of your PRs address the usage of suffixes like `KiB/MiB`, we should get that merged first and then maybe add some of those examples here